.github/workflows/integration-test.yml

```diff
@@ -6,7 +6,7 @@ env:
   TEST_BROWSER_HEADLESS: 1
   CI: 1
   PLUGIN_NAME: opensearch-security
-  OPENSEARCH_INITIAL_ADMIN_PASSWORD: myStrongPassword123!
+  OPENSEARCH_INITIAL_ADMIN_PASSWORD: admin
```

Comment:
Can we use a strong password?

Reply:
Unfortunately since I am shimming in the integration test to test against 2.4, there is no easy way to set a custom admin password (introduced in 2.12). We can probably manually change the admin user's hash, but that seems complex and too much effort just to make this test run with a secure password (in my estimation). Thoughts?

Reply:
Ugh, that stinks but totally understand for 1.3 / 2.4 versions.  Can we have different passwords for different clusters, this would prevent accidents with 2.12+ cluster being mixed up if creds aren't interchangeable

Reply:
IMHO since this is simply spinning up a cluster within a github runner that is spun down upon completion of the test, it is a nice to have but not a blocker to have a strong password. If the consensus is otherwise, maybe we can make two separate jobs, one for older than 2.12 and one for after in order to cleanly test with strong/weak passwords - what does everyone think?

Reply:
We could do that, but I think it might add more confusion (not to forget, the need to update documents). 
@derek-ho  is doing this change to simplify integ testing and as long as the 2.12+ releases dont carry  `admin` as the password, we should be fine.

Reply:
This can be followed separately. I think for now this change should suffice for the intended testing scenarios.